proto/eni.proto

```diff
@@ -0,0 +1,40 @@
+syntax = "proto3";
+
+package dash.eni;
+
+import "types.proto";
+
+enum State{
+    STATE_DISABLED = 0;
+    STATE_ENABLED = 1;
+}
+
+message Eni {
+    string eni_id = 1;
+    bytes mac_address = 2;
+    // Associated Qos profile
+    string qos = 3;
```

Comment:
this shoule be integer

Reply:
I don't think so, here should be the qos name associated to Qos profile.
https://github.com/sonic-net/DASH/blob/main/documentation/general/dash-sonic-hld.md#322-qos
![image](https://github.com/sonic-net/sonic-dash-api/assets/18609639/b0017432-4519-47b3-baec-71690b09eaf1)